### PR TITLE
Fix version mismatch between celery and redis-py

### PR DESCRIPTION
**Pull request by Rohja**
I want to merge this change because...

It fix the issue details in #3830 : Celery can't start because the version of redis-py in Pipenv/Pipenv.lock/requirements.txt is too old.

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by maarcingebala** [comment]
Hey, thanks for opening this PR! Have you tested it locally and can confirm that after these changes Celery works correctly? I'm asking because we're bumping major version of `redis` package from 2.x to 3.x which has some backward incompatibilities.

**Comment by jxltom** [comment]
Which version of celery are you using? I think the problem is Celery instead of redis. As mentioned in https://github.com/mirumee/saleor/pull/3374, redis 3.x is not compatible with celery 4.2.1.

**Comment by Rohja** [comment]
Hi, 
I tested it locally, running `docker-compose up --build` after this change allow celery to run without issue.
I'm running celery 4.2.1 as specified in the original `Pipenv.lock` and `redis:5.0-alpine` from the Docker Hub as specifier in the file `docker-compose.yml`.

**Comment by jxltom** [comment]
As reported in https://github.com/celery/celery/issues/5369, the problem is with celery and kombo. It looks like we should pin kombo < 4.4.0 in Pipfile.

File: requirements.txt [hunk 1]
```diff
@@ -90,7 +90,7 @@ python3-openid==3.1.0 ; python_version >= '3.0'
 pytz==2018.9
 raven==6.9.0
 razorpay==1.2.0
-redis==2.10.6
+redis==3.2.0
```
Comment (jxltom) [inline requirements.txt]: One more thing is that I noticed that botocore and some other packages are also upgraded in Pipfile.lock, could you please update the requirements files by ```pipenv lock -r > requirements.txt``` and ```pipenv lock -r -d > requirements_dev.txt``` as well?

File: Pipfile [hunk 2]
```diff
@@ -58,7 +58,7 @@ social-auth-app-django = ">=2.0.0"
 text-unidecode = "*"
 uwsgi = {version = ">=2.0.0", platform_system = "!= 'Windows'"}
 celery = {version = "*", extras = ["redis"]}
-redis = "<3.0.0"  # this can be safely removed after Celery v4.2.2 is released
+redis = ">=3.2.0"
```
Comment (jxltom) [inline Pipfile]: I would recommend revert this but pin ```kombu = "<=4.3.0"``` to fix this issue.

**Comment by jxltom** [comment]
LGTM 👍

**Comment by maarcingebala** [comment]
@Rohja Thanks for contributing to Saleor!

**Pull request merged**